WORKSPACE

```diff
@@ -109,3 +109,22 @@ http_archive(
 load("@googletest//:googletest_deps.bzl", "googletest_deps")
 
 googletest_deps()
+
+# compile_commands extracts the relevant compile data from bazel into
+# `compile_commands.json` so that clangd, clang-tidy, etc., can use it.
+# Whenever a build file changes, you must re-run
+#
+#   bazel run @hedron_compile_commands//:refresh_all
+#
+# to ingest new data into these tools.
+#
+# See the project repo for more details and configuration options
+# https://github.com/hedronvision/bazel-compile-commands-extractor
+http_archive(
+    name = "hedron_compile_commands",
+    url = "https://github.com/hedronvision/bazel-compile-commands-extractor/archive/3dddf205a1f5cde20faf2444c1757abe0564ff4c.tar.gz",
```

Comment:
small nit: can you add a comment date here? appears to be 2023-05-13

Reply:
I have been copy/pasting this into all my in-progress PRs now that I'm doing this dev in open source, so let me just pull it out into a separate one and we can get it in.

Reply:
Migrated to https://github.com/google/heir/pull/82